.claude/hooks/enforce-sql-migration-pattern.py

```
@@ -0,0 +1,183 @@
+        # Only check SQL files
+        if not file_path.endswith('.sql'):
+            sys.exit(0)
+
+        # Allow operations on valid patch files
+        if is_valid_patch_file(file_path):
+            sys.exit(0)
+
+        # Block modification of protected original SQL files
+        if is_protected_sql_file(file_path):
+            print("\n=== SQL Migration Pattern Enforcer ===", file=sys.stderr)
+            print(f"BLOCKED: Cannot modify original SQL initialization file", file=sys.stderr)
```

Ruff (0.15.1)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)